f8a_report/helpers/ingestion_helper.py

```diff
@@ -0,0 +1,50 @@
+"""Functions to handle worker related tasks."""
+
+import logging
+import os
+import requests
+
+logger = logging.getLogger(__file__)
+
+_APP_SECRET_KEY = os.getenv('APP_SECRET_KEY', 'not-set')
+_INGESTION_API_URL = "http://{host}:{port}/{endpoint}".format(
+    host=os.environ.get("INGESTION_SERVICE_HOST", "bayesian-jobs"),
+    port=os.environ.get("INGESTION_SERVICE_PORT", "34000"),
+    endpoint='ingestions/epv')
+
+
+def ingest_epv(missing_latest_nodes):
+    """
+    Initialize Selinon workers.
+
+    If report has packages which are missing any version from Db, ingestion flow
+    for such packages will be triggered to ingest requested version into Graph DB
+    using Jobs API.
+    """
+    try:
+        result = {}
+        # Iterate through ecosystems.
+        for ecosystem, packages in missing_latest_nodes.items():
+            # Prepare input data in required format.
+            request_data = {
+                'ecosystem': ecosystem,
+                'packages': packages
+            }
+
+            logger.info("Invoking service for triggering ingestion flow for missing_latest_node")
+
+            # Make API call and set token which will be used for authentication.
+            response = requests.post(_INGESTION_API_URL, json=request_data,
```

Comment:
What is the expectation when this request fails. 
We need to have a retry mechanism and if it does not work we need to be able to rerun it when we are able to bring up the job service. 
I think to achieve this, we will need to have a separate cron job which will do this. Otherwise we will have run the daily report run again. 
For immediate needs I think adding a retry loop will be sufficient. There are many libraries that do it. You can look at the tenacity library for this.